Frends.MicrosoftSQL.ExecuteProcedure/Frends.MicrosoftSQL.ExecuteProcedure.Tests/ManualTesting.cs

```diff
@@ -0,0 +1,159 @@
+        GO
+        SELECT * FROM INFORMATION_SCHEMA.TABLES;
+        GO
+   */
+
+    private static readonly string _connString = "Server=127.0.0.1,1433;Database=Master;User Id=SA;Password=Salakala123!";
+    private static readonly string _tableName = "TestTable";
+
+    [TestInitialize]
+    public void Init()
+    {
+        using var connection = new SqlConnection(_connString);
+        connection.Open();
+        var command = connection.CreateCommand();
+
+        //Create
+        command.CommandText = $@"IF NOT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME='{_tableName}') BEGIN CREATE TABLE {_tableName} ( Id int, LastName varchar(255), FirstName varchar(255) ); END";
+        command.ExecuteNonQuery();
+
+        //Insert
+        command.CommandText = $@"CREATE PROCEDURE InsertValues AS INSERT INTO {_tableName} VALUES (1, 'Suku', 'Etu'), (2, 'Last', 'Forst'), (3, 'Hiiri', 'Mikki')";
+        command.ExecuteNonQuery();
+
+        //Select all
+        command.CommandText = $"CREATE PROCEDURE SelectAll AS select * from {_tableName}";
+        command.ExecuteNonQuery();
+
+        //Select single
+        command.CommandText = $"CREATE PROCEDURE SelectSingle AS select * from {_tableName} where Id = 1";
+        command.ExecuteNonQuery();
+
+        //Select lastname
+        command.CommandText = $"CREATE PROCEDURE SelectLastname AS select LastName from {_tableName} where Id = 1";
+        command.ExecuteNonQuery();
+
+        //Update
+        command.CommandText = $@"CREATE PROCEDURE UpdateValue AS update {_tableName} set LastName = 'Edit' where Id = 2";
+        command.ExecuteNonQuery();
+
+        //Delete
+        command.CommandText = $"CREATE PROCEDURE DeleteValue AS delete from {_tableName} where Id = 2";
+        command.ExecuteNonQuery();
+
+        connection.Close();
+        connection.Dispose();
+    }
+
+    [TestCleanup]
+    public void CleanUp()
+    {
+        using var connection = new SqlConnection(_connString);
+        connection.Open();
+        var command = connection.CreateCommand();
+        command.CommandText = $@"IF EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME='{_tableName}') BEGIN DROP TABLE IF EXISTS {_tableName}; END";
+        command.ExecuteNonQuery();
+        command.CommandText = $@"DECLARE @procedureName varchar(500)
+DECLARE cur CURSOR
+      FOR SELECT [name] FROM sys.objects WHERE type = 'p'
+      OPEN cur
+
+      FETCH NEXT FROM cur INTO @procedureName
+      WHILE @@fetch_status = 0
+      BEGIN
+            EXEC('DROP PROCEDURE ' + @procedureName)
+            FETCH NEXT FROM cur INTO @procedureName
+      END
+      CLOSE cur
+      DEALLOCATE cur";
+        command.ExecuteNonQuery();
+        connection.Close();
+        connection.Dispose();
+    }
+
+    // Add following line 'throw new Exception();' under 'dataReader = await command.ExecuteReaderAsync(cancellationToken).ConfigureAwait(false);'
+    [Ignore("Only run manually")]
```

Comment:
I am maybe missing something, but why are we ignoring tests? Also, please add the reason for test ignore to code as a comment, so it is clear to future generations :)